index.html

```diff
@@ -52,114 +52,120 @@ <h1>Personal Profile</h1>
+                    <h2>BikeRacks</h2>
+                    <p class="subDetails">Interactive map</p>
+                    <p><ul>
+                        <li>Helps users find safe bike racks in their area</li>
+                        <li>Interactive voting and filtering of bike racks</li>
+                        <li>Tech used: JavaScript, jQuery, Flask, Python, Firebase, Leaflet</li>
+                    </ul></p>
                 </article>
 				<article>
-					<h2>Austin Community College, Austin, TX</h2>
-					<p class="subDetails">January 2014 - May 2014</p>
-					<p>Earned a Nursing Skills Review Certificate.</p>
+					<h2>CheckItCalendar</h2>
+					  <p class="subDetails">Productivity tracker</p>
+					<p><ul>
+                           <li>Hand-built calendar widget in vanilla HTML5/CSS</li>
+                           <li>Firebase Realtime Database backend</li>
+                           <li>Tech used: JavaScript, jQuery, Firebase, Bootstrap</li>
+                    </ul></p>
 				</article>
 				
-				<article>
-					<h2>Vrije Universiteit Amsterdam, Amsterdam, The Netherlands</h2>
-                    <p class="subDetails">August 2011 - May 2013</p>
-					<p>Completed 87 credits towards a Bachelor of Science in Mathematics.</p>
-				</article>
-                
-                <article>
-					<h2>University of Miami, Coral Gables, FL</h2>
-                    <p class="subDetails">August 2005 - May 2009</p>
-					<p>Earned a Bachelor of Science in Nursing.  Minor focus Spanish Language.</p>
-				</article>
 			</div>
 			<div class="clear"></div>
 		</section>
-		
-		
-		<section>
-			<div class="sectionTitle">
-				<h1>Work Experience</h1>
-			</div>
-			
-			<div class="sectionContent">
+        
+        <section>
+            <div class="sectionTitle">
+                <h1>Skills and Technologies</h1>
+            </div>
+            
+            <div class="sectionContent">
                 <article>
-                    <h2>Dogsitter/Housesitter</h2>
-                    <p class="subDetails">2016 - 2019</p>
-                      <p><ul>
-                          <li>Live-in dogsitter for people who travel a lot for work.</li>
-                      </ul>
-                      </p>
+                    <h2>Programming Languages</h2>
+                    <p>
+                        <ul>
+                            <li>Python, JavaScript,  SQL,  HTML, CSS</li>
+                        </ul>
+                    </p>
                 </article>
                 <article>
-                  <h2>Author</h2>
-                  <p class="subDetails">2017 - 2018</p>
-                    <p><ul>
-                        <li>Writing and freelance editing. Primarily fiction.</li>
-                    </ul>
+                    <h2>Libraries/Frameworks</h2>
+                    <p>
+                        <ul>
+                            <li>React, jQuery, Flask, Firebase, Leaflet, Bootstrap</li>

```

Comment:
If you aren't prepared to talk about react, I'd leave it off of the "things you know" list.

Reply:
Also, why have a list that's one bullet point which is again a list?